src/openforms/registrations/contrib/objects_api/tests/test_plugin.py

```diff
@@ -0,0 +1,32 @@
+from logging import exception
+from django.test import TestCase
+# from openforms.registrations.contrib.zgw_apis.models import ZgwConfig
+from .models import ObjectsAPIConfig
+from ..plugin import ObjectsAPIRegistration as oar
+
+
+class OrcPluginTesterTest(TestCase):
+
+    def setUp(self):
+        # self.plugin = ZGWRegistration("zgw")
+        self.oac = ObjectsAPIConfig.get_solo()
+        drc_client = self.oac.drc_service.build_client()
+        orc_client = self.oac.objects_service.build_client()
+
+        self.lients = [{'type': 'object', 'client': orc_client},
```

Comment:
```suggestion
        self.clients = [{'type': 'object', 'client': orc_client},
```

Did you run these tests locally?